pkg/cmd/util/env.go

```diff
@@ -91,6 +93,32 @@ func ParseEnvironmentArguments(s []string) (Environment, []string, []error) {
 	return env, duplicates, errs
 }
 
+func ParseEnvironmentArgumentsFiles(s []string) (Environment, []string, []error) {
```

Comment:
I can update the existing helper, wasn't sure if such boolean arguments are acceptable style (haven't seen them much in the the codebase).